lonestar/analytics/distributed/bc/README.md

```diff
@@ -1,6 +1,8 @@
 Betweenness Centrality
 ================================================================================
 
+## TODO: add bc-minrounds explanations.
```

Comment:
Check the bc webpage on Galois for mrbc explanation. Also provide a citation to the paper.

Reply:
I'll get around to this in a separate PR.